tests/integration/edge/performance.spec.ts

```
@@ -0,0 +1,824 @@
+        
+        // Current performance should be within 20% of baseline
+        expect(currentTime).toBeLessThan(baselineTime * 1.2)
+      }
+      
+      console.log('Performance baselines established:', Object.entries(baselines).map(([name, time]) => `${name}: ${time.toFixed(2)}ms`).join(', '))
+    })
+  })
+})
+
+// Export helper function for use in other tests
+export { createDomainApiContext }
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)